pkg/cli/cve_cmd_test.go

```diff
@@ -287,6 +288,48 @@ func TestSearchCVECmd(t *testing.T) {
 		So(err, ShouldBeNil)
 	})
 
+	Convey("Test images by CVE ID - positive with retries", t, func() {
+		args := []string{"cvetest", "--cve-id", "aCVEID", "--url", "someURL"}
+		configPath := makeConfigFile(`{"configs":[{"_name":"cvetest","showspinner":false}]}`)
+		defer os.Remove(configPath)
+		mockService := mockServiceForRetry{succeedOn: 2} // CVE info will be provided in 2nd attempt
+		cveCmd := NewCveCommand(&mockService)
+		buff := bytes.NewBufferString("")
+		cveCmd.SetOut(buff)
+		cveCmd.SetErr(buff)
+		cveCmd.SetArgs(args)
+		err := cveCmd.Execute()
+		space := regexp.MustCompile(`\s+`)
+		str := space.ReplaceAllString(buff.String(), " ")
+		t.Logf("Output: %s", str)
+		So(strings.TrimSpace(str), ShouldContainSubstring,
```

Comment:
The function that prints this is defined:
```go
fmt.Fprintln(searchConfig.resultWriter, "[warning] CVE DB is not ready [", i, "] - retry in 3 seconds")
```
I was thinking we can define a constant "RetryPeriod" that it would be 3 (seconds) and add to this test the remaining part of the printed string. This would apply to the rest of the tests too

```suggestion
		So(strings.TrimSpace(str), ShouldContainSubstring, 
		"[warning] CVE DB is not ready [ 0 ] - retry in " + strconv.Itoa(RetryPeriod) + " seconds")
```

If you think it's worth adding it